test/unit/uk/gov/hmrc/testuser/controllers/TestUserControllerSpec.scala

```diff
@@ -127,13 +127,22 @@ class TestUserControllerSpec extends UnitSpec with MockitoSugar with WithFakeApp
 
     "return 201 (Created) with the created organisation" in new Setup {
 
+      def withConsistentOrganisationDetails(testOrganisationCreatedResponse1: TestOrganisationCreatedResponse, testOrganisationCreatedResponse2: TestOrganisationCreatedResponse) = {
+        val consistentOrganisationDetails = OrganisationDetails.random()
+        (testOrganisationCreatedResponse1.copy(organisationDetails = consistentOrganisationDetails),
+          testOrganisationCreatedResponse2.copy(organisationDetails = consistentOrganisationDetails))
+      }
+
       given(underTest.testUserService.createTestOrganisation(refEq(createOrganisationServices))(any())).willReturn(testOrganisation)
 
       val result = await(underTest.createOrganisation()(createOrganisationRequest))
 
       status(result) shouldBe CREATED
-      jsonBodyOf(result) shouldBe toJson(TestOrganisationCreatedResponse(user, password, Some(saUtr),
-        Some(nino), Some(mtdItId), Some(empRef), Some(ctUtr), Some(vrn), Some(lisaManagerReferenceNumber)))
+      val actual = jsonBodyOf(result).as[TestOrganisationCreatedResponse]
+      val expected = TestOrganisationCreatedResponse(user, password, null, Some(saUtr),
+        Some(nino), Some(mtdItId), Some(empRef), Some(ctUtr), Some(vrn), Some(lisaManagerReferenceNumber))
+      val (actualWithConsistentOrganisationDetails, expectedWithConsistentOrganisationDetails) = withConsistentOrganisationDetails(actual, expected)
+      actualWithConsistentOrganisationDetails shouldBe expectedWithConsistentOrganisationDetails
```

Comment:
this is confusing.
can we have at the totp: 
val organisationDetails = OrganisationDetails(...)
val testOrganisation = TestOrganisation(..., organisationDetails)
and here
val expected = TestOrganisationCreatedResponse(user, password, organisationDetails, ...)

Same for individuals